packages/components/inputs/rich-text-utils/src/html/html.tsx

```diff
@@ -279,10 +281,7 @@ const deserializeElement = (
     return children.map((child) => jsx('text', attrs, child));
   }
 
-  // each non-empty text node must be wrapped with a paragraph
-  return children.map((child) =>
-    Text.isText(child) && child.text ? wrapWithParagraph(child) : child
-  );
```

Comment:
This is not needed, it was added as a hot fix of a bug, but text nodes can in fact be added without a wrapping paragraph